superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -42,6 +43,28 @@ import { Query } from '../types';
 
 const SEARCH_HEIGHT = 46;
 
+const debounce = (
```

Comment:
Can we use `debounce` from `lodash`? https://github.com/apache/superset/search?q=debounce